packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -195,14 +295,22 @@ export abstract class DeployExecutor<T> extends DeployRetrieveExecutor<T> {
 }
 
 export abstract class RetrieveExecutor<T> extends DeployRetrieveExecutor<T> {
+  public readonly sourceConflictErrorInfoMsg =
+    'SourceConflictError reported.  Use SFDX: Diff File Against Org and SFDX: Diff Folder Against Org to detect and view conflicts in advance of any retrieve operation.';
+
   protected async doOperation(
     components: ComponentSet,
     token: vscode.CancellationToken
   ): Promise<RetrieveResult | undefined> {
+    const projectPath = getRootWorkspacePath();
     const connection = await WorkspaceContext.getInstance().getConnection();
+    const sourceTracking = await SourceTrackingService.createSourceTracking(
```

Comment:
I noticed above you're calling `ensureLocalTracking` on sourceTracking. Do we not care about that here?

Reply:
Good call.  It is inconsistent because we are not actually using STL to determine conflicts on retrieve operations (currently).  It is possible that when conflict handling for retrieve operations is implemented that VSCE will need to call `ensureLocalTracking` or maybe `ensureRemoteTracking` here in the RetrieveExecutor, but for now it is not needed because VSCE doesn't do anything when STL throws the conflict error in a retrieve op anyway.